doc: Update migration docs (#451)
TheThingsNetwork/docs: doc/content/the-things-stack/migrate-to-v3/_index.md

{{< info >}} This guide refers only to The Things Network V2 and The Things Stack Community Edition deployments. The full migration process that covers all V2 and The Things Stack deployments is described <a href="https://www.thethingsindustries.com/docs/getting-started/migrating/migrating-from-v2/" target="_blank">here</a>. {{</ info >}}

Migrating a single end device can be easily done through The Things Stack Community Edition <a href="https://www.thethingsindustries.com/docs/getting-started/console/" target="_blank">Console</a>. This is convenient if you have a small number of devices, so you can migrate them one by one. See [Migrate End Devices using The Things Stack Community Edition Console]({{< ref "/the-things-stack/migrate-to-v3/migrate-using-console">}}) guide for details.

Migrating end devices in bulk can only be performed with the `ttn-lw-migrate` tool. See [Migrate End Devices using the Migration Tool]({{< ref "/the-things-stack/migrate-to-v3/migrate-using-migration-tool">}}) guide for details.

> {{< note >}} We highly recommend to test the migration on a single end device or a small batch of end devices first, just to make sure the migration process is succesfull and that you are getting an expected result. {{</ note >}}

Reviewer: ```suggestion
{{< note >}} We highly recommend to test the migration on a single end device or a small batch of end devices first, just to make sure the migration process is successful and that you are getting an expected result. {{</ note >}}
```

Depending on your use case, you might want to:

- Keep your gateway connected to The Things Network V2 and route your traffic via Packet Broker to The Things Stack Community Edition - if this is the case, we recommend to read the <a href="https://www.thethingsindustries.com/docs/getting-started/migrating/migrating-from-v2/packet-broker-requirements/" target="_blank">Packet Broker Requirements</a>
- Migrate your gateway to The Things Stack Community Edition - see [Migrate Gateways]({{< ref "/the-things-stack/migrate-to-v3/migrate-gateways" >}})

> {{< warning >}} We strongly recommend The Things Network community members to keep their gateways registered on The Things Network V2 for as long as possible, or to agree on performing coordinated migration to The Things Stack, as this will ensure the community does not loose their LoRaWAN coverage. {{</ warning >}}

Reviewer: ```suggestion
{{< warning >}} We strongly recommend The Things Network community members to keep their gateways registered on The Things Network V2 for as long as possible, or to coordinate with the local community members to perform a migration to The Things Stack, as this will ensure that community members still on The Things Network V2 do not lose their LoRaWAN coverage. {{</ warning >}}
```
